<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoaMainWidget</class>
 <widget class="QWidget" name="RoaMainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Relics of Annorath - Installer</string>
  </property>
  <widget class="QWidget" name="background" native="true">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>0</y>
     <width>901</width>
     <height>531</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/background.png);</string>
   </property>
  </widget>
  <widget class="QLabel" name="qlDesc">
   <property name="geometry">
    <rect>
     <x>337</x>
     <y>77</y>
     <width>530</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(216, 216, 216);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Text&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="qwContent" native="true">
   <property name="geometry">
    <rect>
     <x>337</x>
     <y>150</y>
     <width>520</width>
     <height>250</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="qlSubtitel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>590</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #e27a28;</string>
   </property>
   <property name="text">
    <string>Relics of Annorath Alpha</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
   <property name="margin">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QPushButton" name="qbCancel">
   <property name="geometry">
    <rect>
     <x>724</x>
     <y>460</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width:0px;
border-style:solid;
color: #e27a28;</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="qpNext">
   <property name="geometry">
    <rect>
     <x>565</x>
     <y>460</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width:0px;
border-style:solid;
color: #e27a28;</string>
   </property>
   <property name="text">
    <string>Next &gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="qpBack">
   <property name="geometry">
    <rect>
     <x>405</x>
     <y>460</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width:0px;
border-style:solid;
color: #e27a28;</string>
   </property>
   <property name="text">
    <string>&lt; Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="qlPlaceholder">
   <property name="geometry">
    <rect>
     <x>329</x>
     <y>70</y>
     <width>546</width>
     <height>380</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
color: rgb(255, 255, 255);
background-color: rgba(0,0,0,100);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>qlSubtitel</zorder>
  <zorder>qbCancel</zorder>
  <zorder>qpNext</zorder>
  <zorder>qpBack</zorder>
  <zorder>qlPlaceholder</zorder>
  <zorder>qlDesc</zorder>
  <zorder>qwContent</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
